cli/src/validate.rs

```diff
@@ -291,6 +294,23 @@ fn print_codeowners(codeowners: Option<CodeOwners>) {
                 print_validation_level(JunitValidationLevel::Valid)
             );
             println!("    Path: {:?}", owners.path);
+            for report_validation in report_validations {
+                match report_validation.1 {
+                    Ok(validation) => {
+                        let issues = validation.all_issues_owned();
+                        if issues.iter().any(
+                            |i|
+                            i.error_message == JunitReportValidationIssueSubOptimal::TestCasesFileOrFilepathMissing.to_string()
```

Comment:
Converting enums to strings and doing a comparison is a smell—typically it means we aren't managing our types well. In this case it seems like `all_issues`/`all_issues_owned` is serializing information when it should only do that when needed. That's outside the scope of your change here, but I decided I'd help out with a follow-up PR 😃